openlibrary/templates/account/mybooks.html

```diff
@@ -1,73 +1,229 @@
+              $_('View All Lists')
+              <img class="icon-link__image li-count" src="/static/images/icons/right-chevron.svg">
+            </a>
+          </div>
+        </li>
+      </ul>
+
+      <div style="width: 100%; background: white">
+        <p style="height: 30px"></p>
+        <h1 class="details-title" style="padding: 8px; background: white">My Stats</h1>
+        <p></p>
+      </div>
+    $if owners_page:
+      <ul class="sidebar-section">
+        <li>
+          <div class="carousel-section-header">
+            <a data-ol-link-track="MyBooksSidebar|ReadingStats" href="/account/books/already-read/stats">
+              $_('My Reading Stats')
+              <img class="icon-link__image li-count" src="/static/images/icons/right-chevron.svg">
+            </a>
+          </div>
+        </li>
+        <li>
+          <div class="carousel-section-header">
+            <a class="external-link" data-ol-link-track="MyBooksSidebar|LoanHistory" href="https://archive.org/account/?tab=loans#loans-history">
+              $_('Loan History')
+              <img class="icon-link__image li-count" src="/static/images/icons/right-chevron.svg">
+            </a>
+          </div>
+        </li>
+        <li>
+          <div class="carousel-section-header">
+            <a data-ol-link-track="MyBooksSidebar|MyNotes" $('class=selected' if key=='notes' else '') href="/people/$username/books/notes">
+              $_('My Notes')
+              ($counts['notes'])
+              <img class="icon-link__image li-count" src="/static/images/icons/right-chevron.svg">
+            </a>
+          </div>
+        </li>
+        <li>
+          <div class="carousel-section-header">
+            <a data-ol-link-track="MyBooksSidebar|MyReviews" $('class=selected' if key=='observations' else '') href="/people/$username/books/observations">
+              $_('My Reviews')
+              ($counts['observations'])
+              <img class="icon-link__image li-count" src="/static/images/icons/right-chevron.svg">
+            </a>
+          </div>
+        </li>
+        <li>
+          <div class="carousel-section-header">
+            <a data-ol-link-track="MyBooksSidebar|ImportExport" href="/account/import" $('class=selected' if key=='imports' else '')>
+              $_("Import & Export Options")
+              <img class="icon-link__image li-count" src="/static/images/icons/right-chevron.svg">
+            </a>
+          </div>
+        </li>
+        $if owners_page and ctx.user and ctx.user.in_sponsorship_beta():
+          <li>
+            <div class="carousel-section-header">
+              <a href="/people/$username/books/sponsorships" data-ol-link-track="MyBooksSidebar|Sponsorships" $('class=selected' if key == 'sponsorships' else '')>
+                $_('Sponsorships')
+                $ sponsorship_count = ''
+                $if counts.get('sponsorships', None):
+                  $ sponsorship_count = "%d(" % counts['sponsorships'] % ")"
```

Comment:
Somehow this got reverted -- should be
```suggestion
                  $ sponsorship_count = "(%d)" % counts.get('sponsorships')
```